src/main/java/duke/util/Storage.java

```diff
@@ -0,0 +1,102 @@
+package duke.util;
+
+import duke.task.Task;
+import duke.task.ToDo;
+import duke.task.Deadline;
+import duke.task.Event;
+
+import java.io.IOException;
+import java.io.File;
+import java.io.FileReader;
+import java.io.BufferedReader;
+import java.io.FileWriter;
+import java.util.ArrayList;
```

Comment:
I like how the imports are done individually as specified in the coding standard!

Reply:
I think that the importing statements may or may not be inconsistent,  i think you can use the checkstyle to confirm it!